### ISSUE TITLE
Moblog wrong title and no body text

### ISSUE THREAD

**Issue by gfive** [ExpressionEngine/ExpressionEngine#474]
Default template
Default settings Moblog

When I send 
"test heading" as subject (without the quotes)
and body
"Test to find out if it works." (without the quotes)

Running Moblog "check now"

Moblog Successfully checked
Emails Retrieved: 1
Entries Added: 1
Attachments Uploaded: 0

I get as blog entry:
title "Message-Id:Date:To;" instead of "test heading"
and body

"" instead of "Test to find out if it works."

 
**Environment Details:**
 - EE Version: 5.3.2
 - Moblog Version: 3.3.0
 - PHP Version 7.2.29
 - MariaDB: 10.3.22
 - OS: macOS 10.14
 - Web Server: Apache 2.4.43

**Comment by gfive** [comment]
No response, Is EE still alive?

**Comment by gfive** [comment]
Hmm seems EE is DEAD, so why spending any dime or minute? Instead of bumping major versions, for once update and get EE running...

**Comment by intoeetive** [comment]
@gfive I wasn't able to replicate this with the setup you've mentioned. Is there anything else that should be done in order to get this error?
Also, does the message look ok if you fetch it with your mail client or view in web interface?

**Comment by gfive** [comment]
Using Gmail pop3.
Mail receives okay, header and body are okay.

Only in EE no body and header

**Comment by intoeetive** [comment]
@gfive let's please stick conversation to this GitHub issue, which is where it belongs... copying your last comment here:

> I re-enabled moblog… and reentered my gmail pop
> But getting "Invalid Password” while entered the RIGHT password.
> Not sure why this is happening… also enabled Less secure app access at gmail.
> So for now I am not able eto use it…. The moment I get this to work I can give you access so you can see my problem.
> (No body and different header)

Would you be able to try with different (non-Gmail) email account?

**Comment by creativearc-paul** [comment]
I'm showing similar behavior in EE 5.4.

With both gmail.com and a standard Plesk server POP account, the following occurs:

- Mail IS properly checked. e.g. if I write two emails, two emails are pulled
- A channel entry IS created
- Said channel entry DOES have the proper title..

but that's where it stops.

Even with a body field set, it's simply being ignored.

If I enable Allow overrides in email?, and write a message like this:

{field:my_field format="xhtml"}
    Here's a quick picture of the train station this morning on the way to work.
    Notice the guy selling flowers?  He's been there every morning like clockwork for the last 2 years.
    He's always cheerful and actually has some pretty nice flowers.
{/field: my_field}

It still doesn't work...

TL;DR: the body of the email doesn't make it to the body field of the moblog.
